qa/tests/12-keeper-log-rotation.ps1

```
@@ -0,0 +1,165 @@
+
+# Keeper env (children of this shell inherit; keeper reads them directly).
+$env:PI_KEEPER_LOG_MAX_BYTES = "$CAP_BYTES"
+$env:PI_KEEPER_LOG_CHECK_INTERVAL_MS = "$CHECK_MS"
+$env:PI_KEEPER_CAPTURE_PI_OUTPUT = "1"
+$env:PI_KEEPER_PI_CMD = ConvertTo-Json @("node", $mockPi) -Compress
+$env:MOCK_PI_SIDE_LOG = $sideLog
+
+$keeperProc = $null
+function Cleanup {
+    if ($keeperProc -and -not $keeperProc.HasExited) {
+        try { Stop-Process -Id $keeperProc.Id -Force -ErrorAction SilentlyContinue } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)